docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -0,0 +1,124 @@
+---
+title: The History of C# - C# Guide
+description: What did the language look like in its earliest versions, and how has it evolved since?
+keywords: C#, .NET, .NET Core, What's New, C# History
+author: erikdietrich
+ms.author: wiwagn
+ms.date: 2017/08/09
```

Comment:
date format is mm/dd/yyyy. otherwise weird things happen 😄 

![image](https://user-images.githubusercontent.com/12971179/30261560-18c09012-9682-11e7-8119-ed6ba2c1dd48.png)